### PR TITLE
docs(hibernate): updates example of locking management

### PR DESCRIPTION
**Pull request by machi1990**
The provided example was not compiling - access of a non-static entityManager in a static method.

### CODE REVIEW

**Review by FroMage — APPROVED** [review]
Oops, you're right. Good catch!

**Comment by FroMage** [comment]
Hum… apparently this PR has conflicts now?

**Comment by machi1990** [comment]
> Hum… apparently this PR has conflicts now?

I guess it is related to the modification I pushed yesterday about a typo: `findByIfForUpdate` to `findByIdForUpdate`. Which I did not sync to my PR, so it is still in one of the example I provided.

File: docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc [hunk 1]
```diff
@@ -397,7 +397,7 @@ public class PersonRepository implements PanacheRepository<Person> {
     @Inject
     EntityManager entityManager;
 
-    public Person findByIfForUpdate(Long id){
+    public Person findByIdForUpdate(Long id){
```
Comment (machi1990) [inline docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc]: Maybe the conflict was related to this.

**Pull request merged**